components/ProjectView/ProjectView.vue

```diff
@@ -497,6 +504,12 @@ $paginateRoot_PaddingOffset: 3.5rem;
   font-weight: 600;
   text-align: center;
   background-color: white;
+  .clear-all-null-results {
+    background-color: $blackHaze;
+    padding: 0.3rem;
+    @include borderRadius2;
+    cursor: pointer;
+  }
```

Comment:
Just 2 reminders :)

- `@include borderRadius2;` should go at the top of the SCSS block
- We shouldn't use partial pixel values → `0.3 * 16px = 4.8px`. The browser rounds up to `5px` but we shouldn't make it have to do that 😬 in this case to convert `5px` to rems we'd do → `5 / 16 = 0.3125rem`. I know it's kind of awkward to work with `1px = 0.625rem` but you get used to it I promise!

Reply:
And also mooooost of the time we use nicer-looking/feeling since or quarter-rems thankfully (re: `1rem, 3.25rem, 2.5rem`, etc)